cincinnati/src/plugins/internal/node_remove.rs

```diff
@@ -38,6 +40,10 @@ impl NodeRemovePlugin {
 #[async_trait]
 impl InternalPlugin for NodeRemovePlugin {
     async fn run_internal(self: &Self, io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("node-remove".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```